Hardware/Hardware/CardReader/StripeCardReader/DeclineReason+Stripe.swift

```diff
@@ -0,0 +1,90 @@
+/// Maps Stripes many decline codes to our decline reasons
+/// See `https://stripe.com/docs/declines/codes`
+///
+
+import Foundation
+import StripeTerminal
+
+extension DeclineReason {
+    init(with stripeDeclineCode: String) {
+        let stripeDeclineCodeDictionary: Dictionary<DeclineReason, [String]> =
```

Comment:
We should be able to let the compiler infer the type. That being said, if we still wanted to make it explicit, the following is usually preferred:

`let stripeDeclineCodeDictionary: [DeclineReason, [String]]`

Less angled brackets FTW.